test/suite/test_gc01.py

```diff
@@ -87,6 +73,26 @@ def check(self, check_value, uri, nrows, read_ts):
         session.rollback_transaction()
         self.assertEqual(count, nrows)
 
+    def check_gc_stats(self):
+        c = self.session.open_cursor(
+            'statistics:', None, 'statistics=(fast)')
+
+        self.assertGreaterEqual(c[stat.conn.hs_gc_pages_visited][2], 0)
+        self.assertGreaterEqual(c[stat.conn.hs_gc_pages_removed][2], 0)
+        c.close()
+
+    def get_stat(self, stat):
+        stat_cursor = self.session.open_cursor('statistics:', None, 'statistics=(all)')
+        val = stat_cursor[stat][2]
+        stat_cursor.close()
+        return val
```

Comment:
I don't see this being called anywhere in the changes to this file. If it isn't used, it should be removed.

Reply:
I removed the function here and added it to the test files that were using it.